examples/diagnostic_script.py

```diff
@@ -0,0 +1,201 @@
+"""Summarize usage of RKVST and return estate for customer support purposes."""
+
+# pylint: disable=too-many-locals
+
+from os import environ, path, makedirs, remove, getcwd
+from argparse import ArgumentParser
+from json import dump as json_dump
+from datetime import datetime
+from shutil import move, make_archive
+
+from archivist.archivist import Archivist
+from archivist.logger import set_logger
+
+# import archivist.dictmerge
+
+# functions copied below because I was unable to import \
```

Comment:
all these can be removed. We can pair up if necessary to solve the import problems

Reply:
on reflection this whole file can be removed

add a section in the docs directory to explain usage instead

Reply:
Yes, this file is now outdated and the latest updates are in run.py. I will go ahead and remove now that the updates are transferred over.